src/api/Cosmos.ts

```diff
@@ -0,0 +1,376 @@
+import { getEnv } from "../env";
+import { Tendermint34Client } from "@cosmjs/tendermint-rpc";
+import { DecodedTxRaw, decodeTxRaw } from "@cosmjs/proto-signing";
+import { toHex } from "@cosmjs/encoding";
+import { log } from "@ledgerhq/logs";
+import BigNumber from "bignumber.js";
+import network from "../network";
+import { Operation, SignedOperation } from "../types";
+import { patchOperationWithHash } from "../operation";
+import { calculateFee, GasPrice } from "@cosmjs/stargate";
+
+let tmClient;
```

Comment:
Since we're only using http (no websocket), I think you can instantiate the client only once, for example:
```
let tmClient;
const getTendermint34Client = async () => {
  if (!tmClient) {
    tmClient = await Tendermint34Client.connect(defaultRpcEndpoint);
  }
  return tmClient;
};
```

Reply:
Also well done removing all other clients, if they were not needed.
To be checked if the Tendermint one can be removed too maybe?

Reply:
fix in 869fbfb4